BoxContentSDK/BoxContentSDK/Protocols/BOXContentCacheClientProtocol.h

```diff
@@ -0,0 +1,54 @@
+//
+//  BOXContentCacheClientProtocol.h
+//  BoxContentSDK
+//
+//  Created by Scott Liu on 10/13/15.
+//  Copyright © 2015 Box. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@class BOXBookmarkCommentsRequest;
+@class BOXCommentRequest;
+@class BOXCommentAddRequest;
+@class BOXCommentDeleteRequest;
+@class BOXCommentUpdateRequest;
+@class BOXFileCommentsRequest;
+
+@protocol BOXContentCacheClientProtocol <NSObject>
```

Comment:
I do believe so, I see it as either providing the cache for the entire contentSDK or not, if they really don't want to save particular requests then they can implement no-op and return nil for those particular methods.